### DISCUSSION TITLE
erro message from debug.log

### DISCUSSION THREAD

**Discussion by litaobj-mxd** [Chia-Network/chia-blockchain#6349] [Support]
error message from debug.log is below:
.Error with syncing: <class 'ValueError'>Traceback (most recent call last):
  File "chia/consensus/blockchain.py", line 260, in receive_block
  File "chia/full_node/block_store.py", line 102, in add_full_block
  File "aiosqlite/cursor.py", line 69, in close
  File "aiosqlite/cursor.py", line 31, in _execute
  File "aiosqlite/core.py", line 129, in _execute
  File "aiosqlite/core.py", line 102, in run
sqlite3.ProgrammingError: Cannot operate on a closed database.

how to fix it?or it can be ignored?thanks.
(ubuntu 5.11.0-17-generic, 
software ::chia-blockchain_1.1.5_amd64.deb)

**Comment by BrandtH22** [comment]
@litaobj-mxd , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
